apps/src/applab/applab.js

```diff
@@ -687,6 +687,11 @@ Applab.init = function(config) {
     // and sized in drawDiv().
     Applab.setLevelHtml(level.levelHtml || level.startHtml || "");
 
+    if (!!config.level.projectTemplateLevelName) {
+      studioApp.displayAlert('#codeWorkspace', 350, 'warning',
```

Comment:
I guess I meant to do that here, not inside the component.  This (`Applab.init`) is sort of glue code already.  I agree with  not letting the component examine the DOM.